src/config/index.ts

```
@@ -0,0 +1,244 @@
+        case 'cacheExpiration':
+          const cacheExp = parseInt(value, 10);
+          config.cacheExpiration = isNaN(cacheExp) ? 600 : cacheExp;
+          break;
+        case 'allowCreateTable':
+          config.allowCreateTable = value === 'true';
+          break;
+        case 'allowModifyTable':
+          config.allowModifyTable = value === 'true';
+          break;
+        case 'allowDeleteTable':
+          config.allowDeleteTable = value === 'true';
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)